scripts/import-cp-contexts.sh

```diff
@@ -0,0 +1,249 @@
+#!/usr/bin/env bash
+# Copyright 2024 The KubeStellar Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+set -e
+
+
+arg_names=""
+arg_kubeconfig=""
+arg_context=""
+arg_ip_addr=""
+arg_merge=false
+arg_output=""
+arg_verbose=true
+
+
+display_help() {
+  cat << EOF
+Usage: $0 [--kubeconfig <filename>] [--context <name>] [--names <list of names>] [--replace-localhost <address>] [--merge] [-o <filename>] [-V] [-X]
+--kubeconfig <filename>       use the specified kubeconfig
+--context <name>              use the specified context
+--names <name1>,<name2>       comma separated list of KubeFlex Control Planes names to import, instead of default *all*
+--replace-localhost <address> replaces server addresses "127.0.0.1" with <address>
+--merge                       merge the control planes contexts with the existing cluster contexts
+-o <filename>                 specify a different kubeconfig file to save the contexts (- for stdout)
```

Comment:
The result produced is not just contexts, it is a full kubeconfig.

I might swap the order of these two options and describe them with something like the following.

- `-o <filename>`  specify where to write the resulting kubeconfig
- `--merge` the resulting kubeconfig gets merged into the pre-existing contents of the output filename

Reply:
ok